eth/tasks/deploy.ts

```diff
@@ -506,33 +484,19 @@ export async function deployCaptureFacet(
   return contract;
 }
 
-async function deployDiamondCutFacet({}, libraries: Libraries, hre: HardhatRuntimeEnvironment) {
-  const factory = await hre.ethers.getContractFactory('DiamondCutFacet');
+async function deployDiamond(hre: HardhatRuntimeEnvironment) {
```

Comment:
```suggestion
async function deployDiamond({}, hre: HardhatRuntimeEnvironment) {
```
generally these `deploy*` functions have an empty object if they don't need any parameters at the moment